### PR TITLE
Customize cert for accessing the object store

### PR DESCRIPTION
**Pull request by dockerymick**
https://app.zenhub.com/workspaces/engineering-backlog-do-not-delete-604fab62d4b98d00150a2854/issues/stolostron/backlog/19510

@clyang82 Thanks for your patience! When you get an opportunity, may you review this PR?

### CODE REVIEW

File: observability/customize_observability.adoc [hunk 1]
```diff
@@ -13,10 +13,11 @@ Collect logs about new information that is created for observability resources w
 * <<updating-replicas,Updating the _multiclusterobservability_ CR replicas from the console>>
 * <<forward-alerts,Forwarding alerts>>
 * <<customizing-route-cert,Customizing route certification>>
+** <<customizing-certificates-object-store,Customizing certificates for accessing the object store>>
```
Comment (clyang82) [inline observability/customize_observability.adoc]: It is not a sub-item of `Customizing route certification`. using `*` instead of `**`.

**Comment by clyang82** [comment]
overall looks good for me. Just a small comment. Thanks
/lgtm

File: observability/customize_observability.adoc (hunk 1 above)
Comment (dockerymick) [reply observability/customize_observability.adoc]: Thanks for your review and feedback! @clyang82

**Comment by dockerymick** [comment]
Based on @clyang82 comment and my update, I am moving forward to merge this PR

**Pull request merged**